developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Move on Aptos"
+slug: "move"
+sidebar_position: 0
+---
+
+# Move on Aptos
+
+The Aptos blockchain consists of validator nodes that run a consensus protocol that agrees upon the ordering of transactions and their output when executed on the MoveVM. Each validator node translates transactions along with the current blockchain ledger state as input into the VM, which processes this input to produce a changeset or storage delta as output. Once consensus agrees and commits to the output, it becomes publicly visible. In this guide, we will introduce you to core Move concepts and how they apply to developing on Aptos.
+
+## What is Move?
+
+Move is a safe and secure programming language for Web3 that emphasizes scarcity and access control. An assets in Move can be represented by or stored within *resource*. **Scarcity** is enforced by default, as unless otherwise made explicit objects and instances within Move are not copyable. Only those objects that have explicitly been defined at the bytecode layer as *copy* can be duplicated.
```

Comment:
This sentence reads weird, break it down into two.

**Scarcity** is enforced by making objects and instances (variables? remove this?) within Move are not copyable by default.  Only objects that have been explicitly defined at the bytecode layer as *copy* can be duplicated.